mewarpx/mewarpx/diags_store/flux_diagnostic.py

```diff
@@ -845,6 +900,83 @@ def __init__(self, df, area, step_begin=None, step_end=None, dt=None):
             array_dict=array_dict
         )
 
+class FluxDiagFromFile(FluxDiagBase):
+
+    """Load a flux diag from its minimal saved file.
+    This is meant for postprocessing ONLY; it will not initialize things
+    properly for during-run operations.
+    """
+
+    def __init__(self, basedir='diags', fluxdatafile=None,
+                 fluxdatafileformat='fluxes/fluxdata*', fs=None):
+        """Load from fluxdata_XXXXXXXXXX.dpkl and runinfo.dpkl files.
+        Arguments:
+            basedir (str): Base directory of the diagnostic files.
+                Has to contain a runinfo dill-pickled file.
+            fluxdatafile (str): Filename dill-pickled with FluxDiagBase.save().
+            fluxdatafileformat (str): If a specific fluxdatafile is not
+                specified, this naming format will be used to search for the
+                latest fluxdata file in the base directory.
+            fs (s3fs filesystem): Optional S3 filesystem to load data directly
+                from a S3 bucket.
+        """
+        if fs is None:
+            self.open_command = open
+            self.exists_command = os.path.exists
+            self.glob_command = glob.glob
+        else:
+            self.open_command = fs.open
+            self.exists_command = fs.exists
+            self.glob_command = fs.glob
+
+        self.fluxdatafileformat = fluxdatafileformat
+
+        runinfofile = self._get_runinfofile(basedir)
+        if fluxdatafile is None:
+            fluxdatafile = self._get_fluxdatafile(runinfofile, fs)
+
+        with self.open_command(runinfofile, 'rb') as pfile:
+            self.runinfo = dill.load(pfile)
+
+        self.component_list = self.runinfo.component_list
+        self.species_list = [species.name for species in self.runinfo.species_list]
+
+        with self.open_command(fluxdatafile, 'rb') as pfile:
+            dict_to_load = dill.load(pfile)
+
+        self.__dict__.update(dict_to_load)
+
+    def _get_runinfofile(self, basedir):
+        """Function to look for a runinfo.dpkl file in the current directory."""
```

Comment:
I don't really understand the use of this function and the comment is not correct. In addition to needing a better comment, this will fail if `runinfo.dpkl` is present but no flux diag files were found, with a very misleading error.

Possible approach: Make a list of directories = ['diags'] + reversed(sorted(glob command stuff)). Iterate through them to find first one that contains both runinfo and flux diags. If it fails, print out an error message that no directory was found with both files.

Reply:
Actually we don't need any of that since we don't have restart in WarpX that will rename 'diags' so I just removed all the extra functionality. If we implement that sort of diagnostic handling in WarpX restarts it'll be easy to just reference the `metools` version of this function to regain the removed functionality.